products/microos/main.pm

```diff
@@ -63,9 +63,11 @@ sub load_feature_tests {
         loadtest 'console/kubeadm';
     }
     elsif (check_var 'SYSTEM_ROLE', 'container-host') {
+        my $args = OpenQA::Test::RunArgs->new();
+        $args->{runtime} = 'podman';
```

Comment:
in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14231 you used `CONTAINER_RUNTIME` setting, which seems to be also set on the microos failures I looked at. should we use them here too?

Alternatively, forcing podman sounds ok, as it's just the same as was done before (podman_image was hardcoded)

Reply:
yes, better. Done.